marshmallow_dataclass/__init__.py

```diff
@@ -270,6 +270,10 @@ def field_for_schema(
     >>> field_for_schema(Enum("X", "a b c")).__class__
     <class 'marshmallow_enum.EnumField'>
 
+    >>> import marshmallow_union
```

Comment:
You don't need to import `marshmallow_union` here.

Reply:
fixed :)